simulator/simulator.go

```diff
@@ -73,6 +79,23 @@ func Get() (*Simulator, error) {
 	return simulator, nil
 }
 
+// GetWithFixedSeed behaves like Get() expect that all of the internal
+// heirarchy seeds are derived from the input seed.
+func GetWithFixedSeed(seed int) (*Simulator, error) {
+	s, err := Get()
+	if err != nil {
+		return nil, err
+	}
+	for i := 0; i < 64; i++ {
```

Comment:
Is this intended to be deterministically puesdorandom? Could we just use `math/rand` if so?

Reply:
Fixed to use math.rand